docs/cni/kubernetes/VagrantCoreOS.md

```diff
@@ -0,0 +1,149 @@
+<!--- master only -->
+> ![warning](../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.13.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Deploying Calico and Kubernetes on CoreOS using Vagrant and VirtualBox
+
+These instructions allow you to set up a Kubernetes v1.1.4 cluster with [Calico networking][calico-networking] using Vagrant and the [Calico CNI plugin][calico-cni]. This guide does not setup TLS between Kubernetes components.
+
+## 1. Deploy cluster using Vagrant 
+
+### 1.1 Install dependencies
+
+* [VirtualBox][virtualbox] 5.0.0 or greater.
+* [Vagrant][vagrant] 1.7.4 or greater.
+* [Git][git]
+
+<!--- master only -->
+### 1.2 Clone this project
+
+    git clone https://github.com/projectcalico/calico-containers.git
+<!--- else
+### 1.2 Clone this project, and checkout the **release** release
+
+    git clone https://github.com/projectcalico/calico-containers.git
+    git checkout tags/**release**
+<!--- end of master only -->
+    
+### 1.3 Startup and SSH
+
+Edit `calico-containers/docs/cni/cloud-config/node-config.yaml` and uncomment the line in `/etc/hosts` so that the nodes can resolve the hostname `kubernetes-master`.  The line should look like this:
+```
+172.18.18.101   kubernetes-master
+```
+
+Edit `calico-containers/docs/cni/cloud-config/master-config.yaml` and comment the following line in `calico-node.service` to disabled IP-in-IP, which is not needed for this guide.
+```
+ExecStartPre=/opt/bin/calicoctl pool add 192.168.0.0/16 --ipip --nat-outgoing
+```
+
+Change into the directory for this guide:
+  
+    cd calico-containers/docs/cni/kubernetes/vagrant-coreos
+
+Run
+
+    vagrant up
+
+> *Note*: This will deploy a Kubernetes master and a single Kubernetes node.  To run more nodes, modify the value `num_instances` in the Vagrantfile before running `vagrant up`.
+
+To connect to your servers
+* Linux/Mac OS X
+    * run `vagrant ssh <hostname>`
+* Windows
+    * Follow instructions from https://github.com/nickryand/vagrant-multi-putty
+    * run `vagrant putty <hostname>`
+
+### 1.4 Verify environment
+
+You should now have two CoreOS servers - one Kubernetes master and one Kubernetes node. The servers are named calico-01 and calico-02 
+and have IP addresses 172.17.8.101 and 172.17.8.102.
```

Comment:
Where do those IPs come from?  `ip addr` shows my VMs have 172.18.18.101 and 172.18.18.102 (along with a bunch of others).